Resources/Prototypes/_Sunrise/Catalog/ert_uplink_catalog.yml

```
@@ -66,24 +66,47 @@
+  cost:
+    Bluecrystal: 11
+  categories:
+  - NTAutoGun
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 3
+
+- type: listing
+  id: NTUplinkSunriseClothingBackpackDuffelWeaponRifleM28Filled
+  productEntity: ClothingBackpackDuffelWeaponRifleM28Filled
+  icon: { sprite: _Sunrise/Interface/Misc/nt_icons.rsi, state: m28 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)